as.c

```diff
@@ -189,6 +189,18 @@ static inline vm_operand make_operand(vm_env *env, char *line, const char *data)
         op.type = CONST;
         op.value.id = vm_add_const(env, STR, quoted_strdup(data));
         break;
+    case ':':;
```

Comment:
Why  `case ':':;` here?

Reply:
It seems that I cannot declare an variable in switch statement,
and that's one of the solution I found on stackoverflow.

And the another way doing this is by adding  a `{}` to make it a compound statement.

Reply:
After some researching on K&R C, 
I found that using `;` is just to satisfy the requirement in _labeled-statement_ -- a _null statement_
So `{}` will be the proper way for doing this.